### ISSUE TITLE
Update setting isn't used / Too many site requests

### ISSUE THREAD

**Issue by ffuuuuu** [FaserF/ha-deutschebahn#54]
### The problem

I setup the script to monitor just a single connection. Set it to 5 minutes update time and later even 10 minutes but noticed through my pi.hole that the DB mobile site gets accessed like every 5 seconds. This is very abnormal behavior and is almost like a DDOS to the site when used on mass. So this should really be fixed.

### What version of Home Assistant Core has the issue?

2024-9-3

### What version of DeutscheBahn Integration has the issue?

latest

### What type of installation are you running?

Home Assistant Container

**Comment by FaserF** [comment]
Thanks for the Feedback. This is a similar issue to #48 
5 seconds should not be possible, since the limit and minimum from HA itself is 30 seconds. But you are right, it refreshes every 30 seconds despite the fact that something different is set up. 

Do you use multiple connections which may explains the 5 seconds?

**Comment by ffuuuuu** [comment]
> Thanks for the Feedback. This is a similar issue to #48 5 seconds should not be possible, since the limit and minimum from HA itself is 30 seconds. But you are right, it refreshes every 30 seconds despite the fact that something different is set up.
> 
> Do you use multiple connections which may explains the 5 seconds?

Yes, indeed it is very strange. Just a single connection and 3 added times if that makes sense.

**Comment by sonyboypatrick** [comment]
I'm using this integration since some weeks. I have 4 connections, each of them is configured with a default poll level of 2 minutes and if I look into my adguardhome, mobile.bahn.de is querried a lot.

![image](https://github.com/user-attachments/assets/a7f34e75-39ad-4f9d-a07e-34bd98734e45)

There are gaps with nearly 30 seconds between some of the querries

![image](https://github.com/user-attachments/assets/ceb9310c-6e56-4df7-9cb2-3ce7e2885838)
![image](https://github.com/user-attachments/assets/470ef3b6-06e5-4e4f-9865-75d26f9659d4)

but between the gaps, its querried a lot ...

![image](https://github.com/user-attachments/assets/98132b70-104e-4ac4-81e1-290bf26f80e6)


I hope you can identify the bug.
